rest_api/operator_apis/dns-operator-openshift-io-v1.adoc

```diff
@@ -238,12 +238,17 @@ Type::
 |===
 | Property | Type | Description
 
+| `policy`
+| `array (string)`
+| Policy defines the policy to select upstream resolvers. There are 3 values; `Random` is by default and is a policy that implements random upstream selections,`RoundRobin` is a policy that select hosts based on round robin order, and `Sequential` is a policy that select hosts based on sequential order.
+
 | `upstreams`
 | `array (string)`
-| upstreams is a list of resolvers to forward name queries for subdomains of Zones. Upstreams are randomized when more than 1 upstream is specified. Each instance of CoreDNS performs health checking of Upstreams. When a healthy upstream returns an error during the exchange, another resolver is tried from Upstreams. Each upstream is represented by an IP address or IP:port if the upstream listens on a port other than 53. 
+| upstreams is a list of resolvers to forward name queries for subdomains of Zones. Upstreams are randomized when more than 1 upstream is specified. Each instance of CoreDNS performs health checking of upstreams. When a healthy upstream returns an error during the exchange, another resolver is tried from Upstreams. Each upstream is represented by an IP address or IP:port if the upstream listens on a port other than 53.
```

Comment:
Not sure about capitalization of "upstream".

Reply:
There is a diff between Upstreams and upstream. It should be Upstreams.

Reply:
I think backtick should resolve the confustion to diff between upstreams field/key and actual upstream. I admit its confusing for me too